scripts/setup-env-vars.sh

```diff
@@ -12,7 +12,7 @@ else
   export UPS_HOST=bs-local.com
   export UPS_PORT=8089
   
-  export PGHOST=localhost
+  export PGHOST=bs-local.com
```

Comment:
Why was this changed to `bs-local.com`? Postgres is accessed from test script, not from device running in browser stack.

Reply:
at the end `bs-local.com` resolves to `127.0.0.1` which is the same as for `localhost`. For example for keycloak where some operations are performed from the test script and some from the device, they both use the host `bs-local.com`. I mainly wanted to unified everything to one single host but it doesn't make much different